DOCS.md

```diff
@@ -0,0 +1,61 @@
+# Documentation best practices
```

Comment:
Few more additions

1. the generated code should be always "copy-paste"-able, meaning it can be ported easily to a live working application
1. no class names, only layout components